pkg/runner/grpc.go

```diff
@@ -168,7 +176,42 @@ func buildResponses(resps []*dynamicpb.Message) ([]string, error) {
 	return respsStr, nil
 }
 
-func getMethodDescriptor(ctx context.Context, r *gRPCTestCaseRunner, testcase *testing.TestCase) (protoreflect.MethodDescriptor, error) {
+func getMethodDescriptor(ctx context.Context, r *gRPCTestCaseRunner, testcase *testing.TestCase, conn *grpc.ClientConn) (protoreflect.MethodDescriptor, error) {
+	fullname, err := splitFullQualifiedName(testcase.Request.API)
+	if err != nil {
+		return nil, err
+	}
+
+	var dp protoreflect.Descriptor
+	// if fd, ok := r.fdCache.Load(fullname.Parent()); ok {
+	// 	fmt.Println("hit cache",fullname)
+	// 	return getMdFromFd(fd.(protoreflect.FileDescriptor), fullname)
+	// }
+
+	if r.proto.ServerReflection {
+		dp, err = getByReflect(ctx, r, fullname, conn)
+	} else {
+		if r.proto.ProtoFile == "" {
+			return nil, fmt.Errorf("missing descriptor source")
+		}
+		dp, err = getByProto(ctx, r, fullname)
+	}
+
+	if err != nil {
+		return nil, err
+	}
+
+	if dp.IsPlaceholder() {
+		return nil, fmt.Errorf("%s is not found", fullname)
```

Comment:
You can try to use the following one:

`fmt.Errorf("%q is not found", fullname)`